src/modules/user/user.repository.ts

```diff
@@ -250,6 +251,14 @@ export class SequelizeUserRepository implements UserRepository {
     return this.modelUserNotificationTokens.count({ where: { userId } });
   }
 
+  async loginFailed(uuid: User['uuid'], isFailed: boolean): Promise<void> {
+    const update: any = { errorLoginCount: 0 };
+    if (isFailed) {
+      update.errorLoginCount = Sequelize.literal('error_login_count + 1');
```

Comment:
Why? Setting a new number and updating should be enough, this is also business logic (should be in the auth module and the login usecase should call the user's repository to update this field.